.github/workflows/ci.yml

```diff
@@ -542,9 +542,11 @@ jobs:
           - almalinux/8
           - rockylinux/9@4.0.0
         cgroup_driver: [cgroupfs, systemd]
+        runc: [runc, crun]
```

Comment:
(Maybe we only need either Fedora or Rocky 9 for crun?)

Reply:
I'm fine with an exclude, not every permutation adds value

Reply:
done